help/de/Help_ch_Account-Actions.xml

```diff
@@ -1,910 +1,1049 @@
-        one to mark the account as a <guilabel>Placeholder</guilabel>, and one to mark an account
-        <guilabel>Hidden</guilabel>.
+<!-- FIXME source: The dialogue contains two further items "automatische Zinsbuchung" and "Anfangsbestand" -->
+      <para>Unterhalb des Bereichs befinden sich 5 Kontrollkästchen: eines, um anzuzeigen, ob das Konto
+        <guilabel>steuerrelevant</guilabel> ist, eines um das Konto als
+        <guilabel>versteckt</guilabel> zu kennzeichnen. Ein weiteres Kontrollkästchen wenn das
+        Konto als <guilabel>Platzhalter</guilabel> verwendet wird, zusätzlich eine Auswahl, ob für
+        das Konto <guilabel>automatische Zinsbuchungen</guilabel> erfolgen sollen und letztendlich
+        eines, mit der das Konto für die Aufnahme eines <guilabel>Anfangsbestand</guilabel> genutzt
+        wird.
       </para>
 
-      <para>The <guilabel>Tax Related</guilabel> check-box means that an account has been flagged to be included
-        in the Tax Schedule Report. This flag is only displayed on the <guilabel>Edit
-        Account</guilabel> dialog and is set in the <guilabel>Income Tax Information</guilabel>
-        dialog (
+      <para>Das <guilabel>Steuerrelevant</guilabel> Kontrollkästchen bedeutet, dass das Konto ausgewählt
+        wurde, um in einem Steuer-Bericht berücksichtigt zu werden. Dieses Kennzeichen wird hier
+        nur angezeigt und ist im Dialog <guilabel>Optionen Steuerbericht</guilabel> einzustellen (
         <menuchoice>
-          <guimenu>Edit</guimenu><guimenuitem>Tax Report Options</guimenuitem>
+          <guimenu>Bearbeiten</guimenu><guimenuitem>Optionen Steuerbericht</guimenuitem>
         </menuchoice>
-        ). For related reports see the examples like <guilabel>Tax Schedule Report and TXF
-        Export</guilabel> in (<xref linkend="report-regional"/>).
+        ). Der dazu gehörige Bericht ist im Menü
+        <menuchoice>
+          <guimenu>Berichte</guimenu><guimenuitem>Steuer-Bericht &amp; Elster-Export</guimenuitem>
+        </menuchoice>
+        zu finden. Siehe auch <xref linkend="report-regional"/>.
       </para>
 
-      <para>The <guilabel>Placeholder</guilabel> check-box sets an account to read-only status and prevents the
-        addition of transaction data to the account.
+      <para>Das Kontrollkästchen <guilabel>Platzhalter</guilabel> setzt für ein Konto den Status
+        <quote>schreibgeschützt</quote> und verhindert das Hinzufügen von Geschäftsvorgängen zu
+        diesem Konto. So kann zum Einen das Konto zur Gruppierung weiterer untergeorenter Konten
+        genutzt werden und zum Anderen um versehentliche Änderungen an einem
+        <quote>stillgelegten</quote> Konto zu verhindern. Gewöhnlich sollte das Konto dann einen
+        Saldo von 0 aufweisen.
       </para>
 
+      <tip>
+        <para>Enthält ein Konto weitere Unterkonten, dann empfiehlt es sich, dieses als
+          <guilabel>Platzhalter</guilabel> zu kennzeichnen.
+        </para>
```

Comment:
Der Vorteil davon ist, daß es nicht mehr in der Kontenauswahlliste beim Eingeben von Buchungen erscheint, wodurch die Liste deutlich kürzer wird.